extensions/amp-story/0.1/media-pool.js

```diff
@@ -523,6 +633,8 @@ export class MediaPool {
         this.domMediaEls_[oldDomMediaElId],
         'No media element to put back into DOM after eviction.'));
 
+    this.triggerAllocationEvent_(poolMediaEl, /* isAllocated */ false);
```

Comment:
Same question from `swapPoolMediaElementIntoDom_`: why not put this in `deallocateMediaElement_` and `forceDeallocateMediaElement_`?

Reply:
This one should be moved, thanks!